docs/Troubleshooting.md

```diff
@@ -389,3 +390,76 @@ You can continue re-onboarding after using the `--purge` option
 wget https://github.com/Microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent_GA_v1.4.2-124/omsagent-1.4.2-124.universal.x64.sh
 ```
 * Upgrade packages by executing `sudo sh ./omsagent-*.universal.x64.sh --upgrade`
+
+### I see omiagent using 100% CPU
+#### Probable Causes
+A regression in nss-pem package [v1.0.3-5.el7](https://centos.pkgs.org/7/centos-x86_64/nss-pem-1.0.3-5.el7.x86_64.rpm.html)
+caused a severe performance issue, that we've been seeing come up a lot in Redhat/Centos 7.x distributions.
+
+To learn more about this issue, check the following documentation: 
+* Bug [1667121](https://bugzilla.redhat.com/show_bug.cgi?id=1667121) performance regression in libcurl caused by the use of PK11_CreateManagedGenericObject() [rhel-7.6.z]
+
+#### Debugging omiagent high CPU
+Performance related bugs don't happen all the time, and they are very difficult to reproduce.
+If you experience such issue with omiagent you should use the script omiHighCPUDiagnostics.sh which will collect 
+the stack trace of the omiagent when exceeding a certain threshold.
+
+* Download the script
+```
+wget https://raw.githubusercontent.com/microsoft/OMS-Agent-for-Linux/master/tools/LogCollector/source/omiHighCPUDiagnostics.sh
```

Comment:
the package is not in the master branch yet.
https://github.com/microsoft/OMS-Agent-for-Linux/blob/omi-highcpu-collector/tools/LogCollector/source/omiHighCPUDiagnostics.sh

Reply:
once this PR is merged it will work